src/main/java/net/dries007/tfc/objects/items/metal/ItemOreTFC.java

```diff
@@ -77,9 +77,11 @@ public ItemOreTFC(Ore ore)
                     OreDictionaryHelper.register(this, "gem", "lapis");
                     break;
                 case "bituminous_coal":
-                case "lignite":
                     OreDictionaryHelper.register(this, "gem", "coal");
                     break;
+                case "lignite":
+                    OreDictionaryHelper.register(this, "gem", "lignite");
+                    break;
```

Comment:
No, this is intentional. We register ore type items based on what their product is. Using the ore doesn't suit what the purpose of the ore dictionary is.

Reply:
After some more thought, this is probably the better way to handle this case.